src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+
+    private double radius;
+
+    public Circle(double radius, String color) {
+        this.radius = radius;
+        this.color = color;
```

Comment:
use the constructor of the Figure to initialize this variable, variable color make private